### ISSUE TITLE
[CI]  FieldStatsTests.testGeopoint

### ISSUE THREAD

**Issue by davidkyle** [elastic/elasticsearch#24703]
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+5.4+multijob-unix-compatibility/os=centos/95/consoleFull

Reproduces every time on 5.4 with

```
gradle :core:test -Dtests.seed=EA96B41782CFDF18 -Dtests.class=org.elasticsearch.fieldstats.FieldStatsTests -Dtests.method="testGeopoint" -Dtests.security.manager=true -Dtests.locale=th-TH-u-nu-thai-x-lvariant-TH -Dtests.timezone=US/Pacific-New
```

The stack trace is
```
ERROR   2.98s | FieldStatsTests.testGeopoint <<< FAILURES!
   > Throwable #1: java.lang.NullPointerException
   >    at __randomizedtesting.SeedInfo.seed([EA96B41782CFDF18:9294AE937750EA37]:0)
   >    at org.elasticsearch.fieldstats.FieldStatsTests.testGeopoint(FieldStatsTests.java:709)
```

Assigning to Nick K

**Comment by davidkyle** [comment]
Also failed on [5.3](https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+5.3+multijob-unix-compatibility/os=opensuse/157/console) and [5.x](https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+5.x+multijob-unix-compatibility/os=centos/776/console)

**Comment by davidkyle** [comment]
Another failure reported: 
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+5.4+multijob-unix-compatibility/os=amazon/98/console

**Comment by colings86** [comment]
Another failure: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+5.x+java9-periodic/2537/consoleFull

**Comment by nknize** [comment]
On it... thx guys!

**Comment by nknize** [comment]
Resolved by 433c918ac98a9e7cd323a2e6ea3fe60fd156c409
